### PR TITLE
Remove animation of colour of watch

### PR DESCRIPTION
**Pull request by pedro823**
Let's be fair. No one liked this specific animation anyway.

### CODE REVIEW

File: webpack/entry.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import './animateThunder'
 import './animateTime'
```
Comment (AndrewIjano) [inline webpack/entry.js:2]: Shouldn't you delete this instead of `'./animateHeader'`?

**Review by AndrewIjano — CHANGES_REQUESTED** [review]

**Comment by AndrewIjano** [comment]
> Let's be fair. No one liked this specific animation anyway.

Hehe, yeah... We can come up with better ones in the future

File: webpack/entry.js (hunk 1 above)
Comment (AndrewIjano) [reply webpack/entry.js:2]: (this == `import './animateTime'`)

Comment (pedro823) [reply webpack/entry.js]: whoops. Good riddance. Almost broke the entire JS of the page.

**Review by AndrewIjano — APPROVED** [review]
Ok! Now I think everything is fine

**Pull request merged**